### ISSUE TITLE
About training 

### ISSUE THREAD

**Issue by helloful** [megvii-model/HINet#40]
Hi. 
Thank you very much for open source your code. But I used your network model and hyperparameters to train on the GoPro dataset (the training process was written by myself), and the gradient explosion problem occurred. I tried for a long time, but I didn’t find the problem. Do you have any suggestions?

**Comment by mayorx** [comment]
Hi, helloful, 
Thanks for your attention to HINet.
Are the crop size(256x256) and batch size (8x8) consistent with our setting? If not, you could try to reduce the learning rate.

**Comment by ldlshizhu** [comment]
> 您好，您好， 感谢您对 HINet 的关注。 裁剪大小（256x256）和批量大小（8x8）是否与我们的设置一致？如果没有，您可以尝试降低学习率。

您好，感谢您的贡献。我看到gopro数据集的裁剪在论文和gopro.py中设置的是512x512，是我理解的有误吗

**Comment by mayorx** [comment]
@ldlshizhu , 您好
不好意思我的表达可能有些歧义。 我们在训练之前先做了一步数据预处理， 将gopro数据集的720x1280图片 crop成 512x512 的patches。 在数据预处理之后， 在训练过程中， 图片是从512x512的patch再random crop 256x256。

**Comment by ldlshizhu** [comment]
> @ldlshizhu , 您好 不好意思我的表达可能有些歧义。 我们在训练之前先做了一步数据预处理， 将gopro数据集的720x1280图片 crop成 512x512 的patches。 在数据预处理之后， 在训练过程中， 图片是从512x512的patch再random crop 256x256。

非常感谢您的耐心解答，是我粗心了。祝您一切顺利！

**Comment by ldlshizhu** [comment]
> @ldlshizhu , 您好 不好意思我的表达可能有些歧义。 我们在训练之前先做了一步数据预处理， 将gopro数据集的720x1280图片 crop成 512x512 的patches。 在数据预处理之后， 在训练过程中， 图片是从512x512的patch再random crop 256x256。

非常感谢您的耐心解答，是我粗心了。祝您一切顺利！
